Src/main.c

```diff
@@ -757,9 +796,7 @@ optimizations(int nr)
 		printf("spin: no implied semi-colons (pre version 6.3)\n");
 		return 0; /* no break */
 	default:
-		printf("spin: bad or missing parameter on -o\n");
-		usage();
-		break;
+    return -1;
```

Comment:
This error is now handled in the parsing step, but should probably be re-worked because the return value is supposed to be a boolean, and -1 is just a temporary marker value